SnapMD.VirtualCare.ApiModels/UserProfilesResult.cs

```diff
@@ -224,16 +224,13 @@ public string ProfileImage
         public int? Ethnicity { get; set; }
 
         /// <summary>
-        /// Gets a value indicating whether this instance has default required fields (common for all hospitals).
+        /// Gets a value indicating whether this Staff Profile has default required fields (common for all hospitals).
         /// </summary>
         /// <value>
-        /// <c>true</c> if this instance has required fields; otherwise, <c>false</c>.
+        /// <c>true</c> if this staff profile has required fields; otherwise, <c>false</c>.
         /// </value>
-        public bool HasDefaultRequiredFields
```

Comment:
please keep the old field and mark it as obsolete, so it will not break code in other versions / branches.